docs/maintain/iota-core/2.0/docs/how_tos/validator_setup.md

```diff
@@ -0,0 +1,345 @@
+---
+description: Learn how to become a validator.
+image: /img/icons/iota/iota_core.png
+tags:
+  - IOTA Node
+  - IOTA-core Node
+  - Docker
+  - Install
+  - Run
+  - Validator
+  - Staking
+  - macOS
+  - Windows
+  - Linux
+  - how to
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import LinuxCommands from '@site/docs/_admonitions/_linux-commands.md';
+
+# Validator Setup Guide
+
+Validators secure the network by issuing validation blocks and receive Mana as a reward. To become a validator, you need
+to create an account that can issue blocks, and the account needs to stake tokens. This guide explains how to create the
+account and configure the validator plugin of the docker setup.
+The validator plugin will then try to make the account a validator by issuing a candidacy announcement to the network.
+The validator plugin will start issuing validation blocks if selected for the committee.
+
+:::info
+
+Running a validator implies [running a full-node setup](using_docker.md).
+
+:::
+
+Throughout the guide, we will use the CLI wallet from the iota-SDK and the [node docker setup](./using_docker.md).
+
+:::tip notation
+
+This guide uses the following notation. If `Public Key` has a value of `0xff`, then running the
+command `echo {Public Key}` means replacing the variable `{Public Key}` with its value, i.e., running the
+command `echo 0xff`.
+
+:::
+
+## Set Up the CLI Wallet
+
+You can download the latest release of the CLI Wallet from the following link:
+https://github.com/iotaledger/iota-sdk/releases
```

Comment:
Can we link to a specific version? Otherwise it might get hard(er) to find

```suggestion
https://github.com/iotaledger/iota-sdk/releases/tag/cli-wallet-v2.0.0-alpha.1
```

Reply:
I did not put a specific version because it is going to be outdated so quickly when there is a new release, and links like these tend to not get updated. The version you linked is already an older one, the current one is: https://github.com/iotaledger/iota-sdk/releases/tag/cli-wallet-v2.0.0-beta.1.